satpy/readers/_geos_area.py

```diff
@@ -72,6 +72,7 @@ def get_area_extent(pdict):
             coff: Column offset factor
             loff: Line offset factor
             scandir: 'N2S' for standard (N->S), 'S2N' for inverse (S->N)
+            h: Altitude of satellite (m)
```

Comment:
Since we're defining it, can we be more specific? `h` is from the surface of the Earth, right?

Reply:
Yes，Usualy the h is  frome the surface of  the Earth ,but sometime the h is  the height from the center of Earth.It is depend the data from satellite.

Reply:
Do you know what cases use which measurement for `h`?

Reply:
Looks like you resolved this conversation @longtsing, but you never answered my question. Basically I'm wondering if we know enough information in this function to be more specific in this documentation about what type of `h` is being provided (from center of Earth or from the surface or both)?

Reply:
The docstring for `make_ext`, where this `pdict["h"]` gets passed to, defines https://github.com/pytroll/satpy/blob/2ae2935ee445b9dde36bd74a71bf26c809b48a61/satpy/readers/_geos_area.py#L52

which is also in line with the proj definition https://proj.org/en/9.4/operations/projections/geos.html#cmdoption-arg-h . -> So I would be surprised if there are cases where the height above the Earth center needs to be used, instead of the height above surface.

Btw, we could also specify this further here actually  https://github.com/pytroll/satpy/blob/2ae2935ee445b9dde36bd74a71bf26c809b48a61/satpy/readers/_geos_area.py#L122

Reply:
```suggestion
            h: Altitude of satellite above the Earth's surface (m)
```